spec.md

```diff
@@ -1228,15 +1227,17 @@ when c :
   w <= a
 ```
 
-can be rewritten equivalently as follows:
+A FIRRTL compiler is free to optimize this to the following circuit.
```

Comment:
"by assuming that `w` takes on the value of `a` when `not(c)`"

Reply:
Not sure about the exact wording, but expanding on how we reason about this optimization would be helpful, especially in the context of the previous definition which states that "a FIRRTL Compiler is free to choose any value for an indeterminate value when resolving last connect semantics"